gwu-libraries/orcid-integration
# T1 persist tokens

To test:
- Go through the permissions-granting process via `/?sso`
- Confirm that the corresponding row has been inserted into the database.   Recommend `docker exec -it 123 /bin/bash` into the db container.  Then `psql -U orcidig` and `select * from token;`.

You may also wish to test removing all of the docker images, rebuilding, and confirming that [as long as you leave contents of `/data` alone] the database still contains prior contents (it should).

One issue I haven't fully dealt with is permissions on the `/data` directory.   I seem to need to grant 777 in order for docker to access and write to it as needed.  We can discuss.
